core/Names.cc

```diff
@@ -121,6 +124,10 @@ string Name::show(const GlobalState &gs) const {
                 return absl::StrCat(this->unique.original.data(gs)->show(gs), " (overload.", this->unique.num, ")");
             } else if (this->unique.uniqueNameKind == UniqueNameKind::MangleRename) {
                 return fmt::format("{}${}", this->unique.original.data(gs)->show(gs), this->unique.num);
+            } else if (this->unique.uniqueNameKind == UniqueNameKind::OpusEnum) {
+                // Mentioning this explicitly for grepping: the entire goal of UniqueNameKind::OpusEnum is
+                // to have Name::show print the name as if on the original name, so that our OpusEnum
+                // DSL-synthesized class names are kept as an implementation detail.
```

Comment:
Could you modify this comment to state explicitly that you're falling through here?